analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
+
+\textbf{P-101}: User interface cone search on the object catalog, review the results overplotted on the HIPS map.
+
+\textbf{P-102}: Astronomical Data Query Language (ADQL) cone search on the object catalog with magnitude constraints, create a color-magnitude diagram in the results view.
+
+\textbf{P-103}: Create exploratory color plots of different trans-Neptunian object (TNO) populations and perform statistical analyses to determine potential correlations. 
+
+\textbf{P-201}: Perform an LSST light curve classification (e.g., stars vs quasars vs AGNs) using Portal tools, leveraging visualization features to distinguish populations.
+
+\textbf{P-301}: Query annual data releases and deepCoadds for overlaps with other datasets, and generate publication-quality plots of data and images.
+
+\textbf{P-302}: Query survey property maps and cross-correlate them with shape and photo-z catalogs to investigate cosmic shear. Perform advanced filtering to optimize query performance.
```

Comment:
AFAIK it's not possible to cross-correlate survey property maps in the Portal. AFAIK the survey property maps are butler based and not available via the Portal at all, did you find them accessible?

Reply:
You're right, I think this was intended to be part of the Notebook Aspect section.

Reply:
Confirmed resolved.